### ISSUE TITLE
requires double click on tabs after interaction with Child Interactable Component

### ISSUE THREAD

**Issue by jasan-s** [react-native-community/react-native-tab-view#232]
```
"react": "16.0.0-alpha.6",
"react-native": "0.43.3",
using react-native-tab-view directly from github
```

I currently am using [Wix Interaction Lib](https://github.com/wix/react-native-interactable) component as a child to one of the tab views. However If I interact with the intractable view in any way , the tabs require 2 Clicks before changing Views. IF I don't interact with the Intractable View, the tabs work as normal(changing on one click.) However, Swiping between Tabs always works even after interacting with the Wix  Interaction Component. And After swiping of any kind the Click functionality goes back to normal and start changing on single clicks.

**Comment by jacobbodkin** [comment]
I am having this issue as well. Sub-components will not update props until a slide action (or a change tab action). I have confirmed the _render function in the TabViewAnimated is the stuck point.

**Comment by satya164** [comment]
@jasan-s tabs are normal touchables, are you having problems with other touchables such as `TouchableNativeFeedback` and `TouchableOpacity`?

@jacobbodkin that's unrelated to this issue and already documented under caveats https://github.com/react-native-community/react-native-tab-view#caveats

**Comment by jasan-s** [comment]
@satya164, I'll have to check what the wix interactable component uses underneath. You think the touchable it uses may be the cause.

**Comment by jasan-s** [comment]
Can't seem to pinpoint the cause for this.  If you have time, in order to reproduce the issue.  Return the following component from any of the tabs. 
```import React, { Component } from 'react'
import { StyleSheet, View, Button } from 'react-native'
import Interactable from 'react-native-interactable'

export default class SnapTo extends Component {
  constructor(props) {
    super(props)
    this.state = {
      snapToIndex: 0,
    }
  }
  render() {
    return (
      <View style={styles.container}>
        <Interactable.View
          ref='headInstance'
          snapPoints={[
            { x: -140, y: -250 },
            { x: 140, y: -250 },
            { x: -140, y: -120 },
            { x: 140, y: -120 },
            { x: -140, y: 0 },
            { x: 140, y: 0 },
            { x: -140, y: 120 },
            { x: 140, y: 120 },
            { x: -140, y: 250 },
            { x: 140, y: 250 },
          ]}
          dragWithSpring={{ tension: 2000, damping: 0.6 }}

          initialPosition={{ x: 0, y: 0 }}>
          <View style={{ width: 70, height: 70, backgroundColor: 'red', borderRadius: 35 }} />
        </Interactable.View>
        <View style={styles.button}>
          <Button title='Snap To Next' onPress={this.onButtonPress.bind(this)} />
        </View>
      </View>
    )
  }
  onButtonPress() {
    this.refs.headInstance.snapTo({ index: this.state.snapToIndex })
    this.setState({
      snapToIndex: (this.state.snapToIndex + 1) % 10,
    })
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: 'white',
  },
  button: {
    position: 'absolute',
    top: 110,
    left: 110,
    backgroundColor: 'yellow',
  },
})```

**Comment by karanjthakkar** [comment]
@jasan-s Can you put this in a working sample on snack.expo.io? It would help us in easily reproducing and finding a fix for the issue.

**Comment by jasan-s** [comment]
@karanjthakkar never used expo.io before. I tried adding `import Interactable from 'react-native-interactable'`  I get the error trying to import unknown module. Not sure how to install it so the expo.io can have access.

**Comment by jasan-s** [comment]
@satya164 @karanjthakkar any chance you had time to troubleshoot the issue?

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
